ci/setup-build-test-gdk.ps1

```diff
@@ -3,25 +3,38 @@ param(
     [string] $gcs_publish_bucket = "io-internal-infra-unreal-artifacts-production/UnrealEngine",
     [string] $msbuild_exe = "${env:ProgramFiles(x86)}\Microsoft Visual Studio\2019\BuildTools\MSBuild\Current\Bin\MSBuild.exe",
     [string] $build_home = (Get-Item "$($PSScriptRoot)").parent.parent.FullName, ## The root of the entire build. Should ultimately resolve to "C:\b\<number>\".
-    [string] $unreal_engine_symlink_dir = "$build_home\UnrealEngine"
+    [string] $unreal_engine_symlink_dir = "$build_home\UnrealEngine",
+    [string] $gym_version_path = "$unreal_engine_symlink_dir\UnrealGDKTestGymVersion.txt"
```

Comment:
As mentioned in the other PR, I think it will be easier if this file lives in the GDK.